src/java.base/share/classes/java/util/zip/ZipCoder.java

```diff
@@ -133,29 +140,6 @@ boolean hasTrailingSlash(byte[] a, int end) {
             Arrays.mismatch(a, end - slashBytes.length, end, slashBytes, 0, slashBytes.length) == -1;
     }
 
-    // Implements normalizedHash by decoding byte[] to char[] and then computing
-    // the hash. This is a slow-path used for non-UTF8 charsets and also when
-    // aborting the ASCII fast-path in the UTF8 implementation, so {@code h}
-    // might be a partially calculated hash code
-    int normalizedHashDecode(int h, byte[] a, int off, int end) {
```

Comment:
The comment is misleading after JDK-8260010 since the UTF8 implementation can't use this method, and I should have rewritten or removed it with JDK-8260010.